src/main.rs

```diff
@@ -96,6 +104,18 @@ async fn main() -> Result<()> {
             let review = prr.get_pr(&owner, &repo, pr_num, force).await?;
             println!("{}", review.path().display());
         }
+        Command::Edit { pr, force } => {
+            let (owner, repo, pr_num) = prr.parse_pr_str(&pr)?;
+            let review = prr.get_pr(&owner, &repo, pr_num, force).await?;
+            let editor = env::var("EDITOR")?;
+            let path = review.path();
+            std::process::Command::new(editor)
+                .arg(path)
+                .spawn()
+                .expect("Failed to execute editor process")
+                .wait()
+                .expect("Editor returned non-zero status");
```

Comment:
No panics please -- prefer returning `Result` instead:

```suggestion
                .spawn()
                .context("Failed to execute editor process")?
                .wait()
                .context("Editor returned non-zero status")?;
```

Reply:
Thanks--I don't write Rust very frequently.